Tutorials/Communication_Protocols.md

```diff
@@ -215,12 +217,14 @@ Finally, the master will issue a stop condition on the bus by pulling SCL high,
 It's a little complicated, but the Tessel takes care of all the details for you. Using the I2C pins on port A looks like this:
 
 ```js
-var tessel = require('tessel'); // import tessel
-var portA = tessel.port['A']; // use Port A
-var slaveAddress = 0xDE; // This is the address of the attached module/sensor
-var i2c = new portA.I2C(slaveAddress)
-
-i2c.send(new Buffer([0xde, 0xad, 0xbe, 0xef]), function (error) {
+const tessel = require('tessel');
+// Unpack port A from tessel.port and name it "port"
+const { A: port } = tessel.port;
```

Comment:
This is also new to me. Does it mean something different than the previous notation?